web/wp-content/themes/dctx-v2/index.php

```diff
@@ -0,0 +1,55 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-w5jv6v.php, but found phpcs-scan-w5Jv6V.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).